src/sammich/plugins/contributors.py

```diff
@@ -46,36 +46,63 @@ def format_data(prs, issues, comments):
             user_data[user] = {"prs": 0, "issues": 0, "comments": 0}
         user_data[user]["comments"] += 1
 
-    max_name_length = max(len(user) for user in user_data.keys())
-    max_prs_length = max(len(str(counts["prs"])) for counts in user_data.values())
-    max_issues_length = max(len(str(counts["issues"])) for counts in user_data.values())
-    max_comments_length = max(len(str(counts["comments"])) for counts in user_data.values())
-
-    table = [
-        f"| {'User':<{max_name_length}} | PRs Merged | Issues Resolved | Comments |",
-        f"|{'-' * (max_name_length + 2)}|:{'-' * (max_prs_length + 2)}:|"
-        f":{'-' * (max_issues_length + 2)}:|:{'-' * (max_comments_length + 2)}:|",
-    ]
+    # Calculate column widths with minimum lengths
+    max_user_length = max(len(user) for user in user_data.keys())
+    max_prs_length = max(max(len(str(counts["prs"])) for counts in user_data.values()), 10)
```

Comment:
Could you explain the `max(max(len(str(` logic here?
Thanks!

Reply:
basically sir ,this one is for count how much space we should use in each column